scipy-1.0/paper.tex

```diff
@@ -218,6 +229,62 @@ \section*{Discussion}
 \textbf{Future development}.
 \textit{This section should include key issues: sparse arrays, ndimage pixel vs point, splines, fftpack vs. np.fft and linalg vs. np.linalg, under-maintained submodules.}
 
+\subsection*{ndimage pixel vs point}
```

Comment:
From `master`:

> The Discussion should be succinct and must not contain subheadings.

Is that part of the template or just an editor recommendation? If it's part of the template, we should honor it.

Reply:
I propose breaking Future Work into its very own subheading.

Reply:
those were the instructions in the template, we should try to adhere to them

Reply:
That’s exactly what I’m worried about. This section is a subsection of Discussion, has headings, and so forth. Either we need to compress it significantly or put Future work itself into its own subsection, outside of discussion.